doc/source/ray-overview/examples.rst

```diff
@@ -136,13 +136,6 @@ Ray Examples
 
         Perform batch forecasting on NYC Taxi Dataset with Prophet, ARIMA and Ray AIR
 
-    .. grid-item-card:: :bdg-secondary:`Code example`
-        :class-item: gallery-item
-        :link: /ray-air/examples/gptj_deepspeed_fine_tuning
```

Comment:
@richardliaw Can you have a look if we actually intended to remove this? If no we should bring it back with the updated link :)

Reply:
This is intended, there was a duplicate: https://github.com/peytondmurray/ray/blob/918de4f5a15d3bfd7aaaf7c7ff4d83d067a1806f/doc/source/ray-overview/examples.rst?plain=1#L63.